includes/Article.php

```diff
@@ -218,6 +218,14 @@ public function getContent() {
 			return $text;
 		} else {
 			$this->fetchContent();
+			// Wikia: Temporary Investigation for PLATFORM-1355
+			if(empty($this->mContent)) {
+					global $wgCityId;
+					Wikia\Logger\WikiaLogger::instance()->error( __METHOD__ . ' empty content PLAT1355', [
+						'wiki_id' => $wgCityId,
```

Comment:
Thanks! I've seen it logged in other places, so that may be a legacy.  I'll remove it.